### ISSUE TITLE
Fold https://readme.drone.io/ into http://docs.drone.io/

### ISSUE THREAD

**Issue by settermjd** [drone/docs#375]
To avoid any confusion, these two domains should be combined into one, ideally docs.drone.io, and the GitHub for readme.drone.io archived or deleted. That way, there's one central focal point for users and contributors to come to.

**Comment by tboerger** [comment]
I think that's the plan when 0.9 gets released.

**Comment by bradrydzewski** [comment]
yes, readme.drone.io represents the future version of the docs. When 0.9 is released I will redirect docs.drone.io to readme.drone.io, and then going forward we can archive documents by version using a subdirectory (/0.9 /0.10, etc)
